src/app.js

```diff
@@ -0,0 +1,27 @@
+var bodyParser = require('body-parser');
+var express = require('express');
+
+var maryStatus = 418;
+var maryResponse = {im: 'a teapot'};
+
+var app = express();
+
+app.use(bodyParser.json());
+
+app.get('/alligators/Mary', function (request, response) {
+    response.set('Content-Type', 'application/json; charset=utf-8');
+    response.status(maryStatus).send(maryResponse);
+});
+
+app.post('/dodgy-backdoor/control-what-mary-does', function (request, response) {
```

Comment:
I thought adding a dodgy back door to the app was what we were trying to avoid by using a provider state server?